### PR TITLE
Lenovo, Redmi in list now

### PR DESCRIPTION
**Pull request by RohanSakhale**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-  - "5.3"
```
Comment (mimmi20) [inline .travis.yml]: Please revert this. This is fixed in PR #680.

File: Mobile_Detect.php [hunk 2]
```diff
@@ -94,13 +94,13 @@ class Mobile_Detect
      * HTTP headers in the PHP-flavor. So HTTP_USER_AGENT and SERVER_SOFTWARE.
      * @var array
      */
-    protected $httpHeaders = array();
+    protected $httpHeaders = [];
```
Comment (mimmi20) [inline Mobile_Detect.php]: Please keep this file compatible to PHP 5.3 an use the long array syntax.

Comment (RohanSakhale) [reply Mobile_Detect.php]: Ok, will do the changes as required to support PHP 5.3

**Comment by RohanSakhale** [comment]
@serbanghita, could you please re-review the changes now

**Comment by RohanSakhale** [comment]
@serbanghita the break seen above is due to squizlabs/php_codesniffer that requires php 5.4 and above, please suggest if we can drop version of this library.

**Comment by serbanghita** [comment]
@RohanSakhale drop it
I haven't got time to review yet

**Pull request closed without merging**